docs/resources/samples/convert-csv.md

```diff
@@ -0,0 +1,88 @@
+1. Rename the **Get file content** step by going the **...** menu of that step (in the upper right corner of the connector) and selecting the **Rename** option. Change the step name to "Get Excel template".
+
+     :::image type="content" source="../../images/convert-csv-flow-1.png" alt-text="The completed OneDrive for Business connector in Power Automate, renamed to be Get Excel template.":::
+1. Get all the files in the "output" folder. Add a **New step** that uses the **OneDrive for Business** connector and the **List files in folder** action. Provide the folder path that contains the .csv files.
+    * **Folder**: /output
+
+    :::image type="content" source="../../images/convert-csv-flow-2.png" alt-text="The completed OneDrive for Business connector in Power Automate.":::
+1. Add a condition so that the flow only operates on .csv files. Add a **New step** that is the **Condition** control. Use the following values for the **Condition**.
+    * **Choose a value**: *Name* (dynamic content from **List files in folder**). Note that this dynamic content has multiple results, so an **Apply to each** *value* control surrounds the **Condition**.
+    * **ends with** (from the dropdown list)
+    * **Choose a value**: .csv
+
+    :::image type="content" source="../../images/convert-csv-flow-3.png" alt-text="The completed Condition control with the Apply to each control around it.":::
+1. The rest of the flow is under the **If yes** section, since we only want to act on .csv files. Get an individual .csv file by adding a **New step** that uses the **OneDrive for Business** connector and the **Get file content** action. Use the **Id** from the dynamic content from **List files in folder**.
+    * **File**: *Id* (dynamic content from the **List files in folder** step)
+1. Rename the new **Get file content** step to "Get .csv file". This helps distinguish this file from the Excel template.
+1. Make the new .xlsx file, using the Excel template as the base content. Add a **New step** that uses the **OneDrive for Business** connector and the **Create file** action. Use the following values.
+    * **Folder Path**: /output
+    * **File Name**: *Name without extension*.xlsx (choose the *Name without extension* dynamic content from the **List files in folder** and manually type ".xlsx" after it)
+    * **File Content**: *File content* (dynamic content from **Get Excel template**)
+
+     :::image type="content" source="../../images/convert-csv-flow-4.png" alt-text="The Get .csv file and Create file steps of the Power Automate flow.":::
+1. Run the script to copy data into the new workbook. Add the **Excel Online (Business)** connector with the **Run script** action. Use the following values for the action.
+    * **Location**: OneDrive for Business
+    * **Document Library**: OneDrive
+    * **File**: *Id* (dynamic content from **Create file**)
+    * **Script**: Convert CSV
+    * **csv**: *File content* (dynamic content from **Get .csv file**)
+
```

Comment:
I finished the tutorial and am not sure how to test it. When I try to run a manual test, I get an error. :(

Steps 6 and 7 of the [Cross-reference worksheets tutorial](https://docs.microsoft.com/en-us/office/dev/scripts/resources/samples/excel-cross-reference) direct the user to save the flow and then describe how to test it. Is that something we can include for this tutorial?

Reply:
Added those steps. Can I see the error you're getting?

Reply:
(Sent error screenshot offline)